WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -446,6 +448,30 @@ def submitJobs( self ):
       platform = self.queueDict[queue]['Platform']
       siteMask = siteName in siteMaskList
 
+      # Check the status of the Site
+      result = self.sstClient.getSiteStatuses({siteName})
+      if not result['OK']:
+        self.log.error( "Can not get the status of site %s: %s" % (siteName, result['Message']) )
+        continue
+      if result['Value']:
+        result = result['Value'][siteName]   #get the value of the status
+
+      if result not in ('Active', 'Degraded'):
+        self.log.verbose( "Skipping site %s: site not usable" % siteName )
+        continue
+
+      # Check the status of the ComputingElement
+      result = self.rssClient.getElementStatus(ceName, "ComputingElement")
+      if not result['OK']:
+        self.log.error( "Can not get the status of computing element %s: %s" % (siteName, result['Message']) )
+        continue
+      if result['Value']:
+        result = result['Value'][ceName]['ComputingElement', 'all']   #get the value of the status
```

Comment:
It is necessary because it returns a dictionary like this one below:
```
      >>> getElementStatus('CE42', 'ComputingElement')
          S_OK( { 'CE42': { ('ComputingElement', 'all'): 'Active' } } } )
```